scripts/test

```diff
@@ -18,6 +18,6 @@ if [[ "${BASH_SOURCE[0]}" == "${0}" ]]; then
         usage
     else
         echo "Executing Scala test suite"
-        ./sbt "++${SCALA_VERSION:-2.11.12}" test
+        ./sbt -J-Xmx4G "++${SCALA_VERSION:-2.11.12}" test
```

Comment:
After updating sbt plugins, I began to experience exit code 137. Since we have 4GB of RAM [by default](https://circleci.com/docs/2.0/configuration-reference/#resource_class), I figure this is a good value to pin.

Reply:
Could we export `JVM_OPTS` in the CircleCI configuration so that it applies across CircleCI builds? Alternative, if we don't think it'll work across any environment w/o 4GB of heap, then we can add a `.jvmopts` file to the repo.